src/node_contextify.h

```diff
@@ -10,14 +10,14 @@ namespace contextify {
 
 class ContextifyContext {
  protected:
-  // V8 reserves the first field in context objects for the debugger. We use the
-  // second field to hold a reference to the sandbox object.
-  enum { kSandboxObjectIndex = 1 };
-
   Environment* const env_;
   Persistent<v8::Context> context_;
 
  public:
+  // V8 reserves the first field in context objects for the debugger. We use the
+  // second field to hold a reference to the sandbox object.
+  enum { kSandboxObjectIndex = 1, kAllowWasmCodeGeneration = 2 };
```

Comment:
Having the indices this way would make non-VM contexts have a hole at index 1, which doesn't sound too desirable. Also, ditto on the "core should not depend on contextify" comment here.

In other words:

1. Move `kAllowWasmCodeGeneration` to `node_internals.h`, and set it to 1. Keep `kSandboxObjectIndex` here however.
2. Make `kSandboxObjectIndex` equal to `kAllowWasmCodeGeneration + 1`.

----

You don't have to fix it now (though you certainly could), but "V8 reserves the first field in context objects for the debugger" is no longer true since v8/v8@5f90a6eb067. This means that `kAllowWasmCodeGeneration` could just be 0.